spec.bs

```diff
@@ -825,23 +824,44 @@ Moreover, each {{SharedStorageWorklet}}'s [=global scopes|list of global scopes=
     Issue(144): Store |reportingUrlMap| inside a [=fenced frame reporter=] class associated with |fencedFrameConfigStruct|. Both of these still need to be added to the draft [[Fenced-Frame]].
   </div>
 
+
   ## Entropy Budgets ## {#budgets}
 
   Because [=bits of entropy=] can leak via {{SharedStorageWorklet/selectURL()}}, the [=user agent=] will need to maintain budgets to limit these leaks.
 
-  ### Navigation Entropy Budget ### {#nav-budget}
+  On a call to {{SharedStorageWorklet/selectURL()}}, when any of these budgets are exhausted, the [=default selectURL index=] will be used to determine which URL to select.
+
+  <div algorithm>
+    To <dfn>get the default selectURL index</dfn>, given {{sequence}}<{{USVString}}> |urls|, run the following steps:
+
+    1. Return an {{unsigned long}} from the [=the exclusive range|range=] from 0 to |urls|’s [=list/size=], exclusive in [=implementation-defined=] way so that the returned index is independent from the registered operation class's "`run`" method..
+  </div>
+
+  The <dfn>default selectURL index</dfn> is the index obtained by running [=get the default selectURL index=], given {{sequence}}<{{USVString}}> |urls|.
```

Comment:
Now that the clean-up touches this section, can we just define it to be 0 rather than making it implementation-defined?

Reply:
Done.